src/status_im/utils/ethereum/tokens.cljs

```diff
@@ -414,6 +414,10 @@
                     :name     "Livepeer Token"
                     :address  "0x58b6a8a3302369daec383334672404ee733ab239"
                     :decimals 18}
+                   {:symbol   :OST
+                    :name     "Open Simple Token"
```

Comment:
Name and symbol need to match the values from the smart contract:

```
    string  public constant TOKEN_SYMBOL   = "ST";
    string  public constant TOKEN_NAME     = "Simple Token";
```

Reply:
And [Etherscan](https://etherscan.io/address/0x2c4e8f2d746113d0696ce89b35f0d8bf88e0aeca) shows it as Simple Token (ST) and so does every other tool that relies on what's in the contract.

We have another pending PR with token config validation against the contract fields. It will show an error popup if these don't match. The only exception will be the case when those fields aren't defined in the contract (or are empty), since they are optional. But if it is defined and it's also different, there's a discrepancy.

We're trying to move away from manual lists, exceptions and all that. That's why I raised the issue. Is there any chance that you can still rename `name` and `symbol` fields in the contract or at least remove them?

Reply:
And [Etherscan](https://etherscan.io/address/0x2c4e8f2d746113d0696ce89b35f0d8bf88e0aeca) shows it as Simple Token (ST) and so does every other tool that relies on what's in the contract.

We have another pending PR with token config validation against the contract fields. It will show an error popup if these don't match. The only exception will be the case when those fields aren't defined in the contract (or are empty), since they are optional. But if it is defined and it's also different, there's a discrepancy.

We're trying to move away from manual lists, exceptions and all that. That's why I raised the issue. Is there any chance that you can still rename `name` and `symbol` fields in the contract or at least remove them?

Reply:
True. I updated it to be Simple Token (ST).

Reply:
Cheers. Any hint where we can get some for testing?

Reply:
We can send you some :) Can you give me an address where to? You can send me an email if you want: martin@ost.com

Reply:
Thanks. Sent it